draft-ietf-intarea-provisioning-domains.xml

```diff
@@ -164,9 +179,18 @@
     <section title="Introduction">
       <t>It has become very common in modern networks for hosts to access the
       internet through different network interfaces, tunnels, or next-hop
-      routers. To describe the set of network configurations associated with
-      each access method, the concept of Provisioning Domain (PvD) was defined
-      in <xref target="RFC7556"/>.</t>
+      routers. For example, Alice has a mobile phone provider and a broadband
+      provider in her home, her devices should be capable of seamlessly
+      transitioning from one to the other and be able to use her WiFi to
+      access local ressources or use the more suitable link on a per
```

Comment:
Can `per application` be `per-application`?

Reply:
Done